scanner/dockerfile.go

```diff
@@ -53,5 +53,18 @@ func ScanDockerfile(dockerfilePath string, config *ScannerConfig) (*SourceInfo,
 		s.Port = defaultPort
 	}
 
+	// extract volume - handle both plain string and JSON format, but only allow one path
+	re := regexp.MustCompile(`(?m)^VOLUME\s+(\[\s*")?(\/[\w\/]*?(\w+))("\s*\])?\s*$`)
```

Comment:
It's usually best to define this as a global variable so that the `MustCompile` runs once and panics on startup rather than only if this code path is used.

e.g.

```go
var extractVolumeRegex := regexp.MustCompile(`(?m)^VOLUME\s+(\[\s*")?(\/[\w\/]*?(\w+))("\s*\])?\s*$`)
```

Reply:
This is a regular expression that will rarely be used (only if the scanner is Dockerfile), and even then, will only be used once.